### PR TITLE
Card Component Aadded

### PR DESCRIPTION
**Pull request by sameep-s**
What's added: Different styles of cards are added into the library namely.

You can view the preview [here](https://deploy-preview-4--aphro-css.netlify.app/components/card.html).

1. Cards with badges
2. Cards with dismiss
3. Cards with text overlay
4. Text only cards
5. Vertical/Horizontal cards

### CODE REVIEW

File: css/comp.css [hunk 1]
```diff
@@ -99,6 +99,7 @@
 
 .slider {
   width: 100%;
+}
 /* ----------------------------------------- Simplified Grid */
 
 .row-ex {
```
Comment (arjunbharti) [inline css/comp.css]: Write proper name for class. It enhances code readability.

**Review by arjunbharti — COMMENTED** [review]
UI looks clean and decent. Just added a comment for some minor change.

File: css/comp.css (hunk 1 above)
Comment (sameep-s) [reply css/comp.css]: sure. i'll keep that in mind.

**Review by ajit-sharmaGH — APPROVED** [review]
good to go

**Pull request merged**